### ISSUE TITLE
:bug: `/version/:version/extensions/official` returns 500 for pre-release Talos versions

### ISSUE THREAD

**Issue by Unix4ever** [siderolabs/image-factory#75]

**Comment by smira** [comment]
I don't think this is accurate, it doesn't:

```shell
curl https://factory.talos.dev/version/v1.6.0-alpha.1/extensions/official
```

It might return this for _some_ versions, but not all of them.

**Comment by Unix4ever** [comment]
Right.
Versions list:
```
1.3.0 - 1.3.7
1.4.0-alpha.0 - 1.4.0-beta.1
1.5.0-alpha.0 - 1.5.0-alpha.3
```

**Comment by smira** [comment]
I don't think we should support alpha/beta versions for old releases which came before Image Factory even existed, but I will look into others. In either case, we can make it empty on Image Factory side as well.

**Comment by smira** [comment]
Fixed up 1.5.0-alphas, but I will drop support for any alpha/beta versions except for the latest one (i.e. 1.6.0).
